core/translate/group_by.rs

```diff
@@ -373,6 +393,24 @@ pub fn emit_group_by<'a>(
             .expr_to_reg_cache
             .push((expr, reg_group_exprs_acc + i));
     }
+
+    // Offset for the next expressions after group_by
```

Comment:
`// Register offset for the non-aggregate expressions that are not part of GROUP BY`